### ISSUE TITLE
vocab_size in tokenizer for Citrinet

### ISSUE THREAD

**Issue by MelikaBahmanabadi** [NVIDIA/NeMo#3057]
Hi,
I used **stt_en_citrinet_1024** as a based model for fine-tuning with the Persian audio dataset. In your idea what is the best tokenizer for this? (**wpe** or **spe**). if **spe** is good what is the better type? (**bpe**, **unigram**, **char**, **word**). Also, I want to know about the vocab_size. What is the maximum number of vocab_size for tokenizer? If the vocab_size is increased, so the WER decreased?
Thanks

**Environment overview**

    Environment location: Google Colab
    Method of NeMo install: !pip install nemo_toolkit[asr]
    NeMo version: r1.2.0
    Learning Rate: 1e-2

**Environment details**

    OS version : "Ubuntu20.04.3 LTS"
    PyTorch version : "1.9.1"
